<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>第一个操作数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNum1"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxOper">
        <item>
         <property name="text">
          <string>+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>第二个操作数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNum2"/>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>结果</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditResult"/>
      </item>
     </layout>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
